EIPS/eip-4974.md

```diff
@@ -1,7 +1,7 @@
 ---
 eip: 4974
-title: Experience (EXP) Token Standard
-description: A standard interface for fungible, non-tradable tokens, also known as EXP.
+title: Rating
+description: A standard interface for ratings.
```

Comment:
I do not believe that "Rating" is the best description for this. How about "Fungible Non-Transferrable Tokens" (but dropping the EXP)?

Reply:
Fungibility as a choice of terminology since it describes "interchangeability." But how can there be fungibility if the tokens cannot be traded?

Reply:
Interchangeability refers to equality between tokens. Basically, it means you can stick a number on some quantity and say "I have X amount of Y," instead of saying "I have the Y." For example, if you have 3 apples, you can say "I have 3 apples," but you can't say "I have the apples." You do not have all the apples in the world (unless there is a _really_ bad shortage of them).

Likewise, the description of this standard means it is fungible. You have a certain _amount_ of the non-tradeable token, not just zero or one. This makes it fungible.

Reply:
I'm no economist but I had understood fungibility to be a trait of the goods exchange or substitution quality. The definition is that you can use either this or that gold/oil because it doesn't matter: it's fungible - individual units of itself can be used as substitution interchangably.

I wanna say that I agree that e.g. the experience points players gain in an MMORPG are comparable. It's not that everyone gains experience points on their personal scale. Still my interpretation is that fungibility only exists within the world of trading.

I know "fungible" is in everyone's hearts and minds. But e.g. the more accurate description IMO would be "Comparable Non-Transferable Tokens"

Reply:
I'm fine with "Comparable Non-Transferable Tokens".

Reply:
Makes sense, but the focus of this EIP is sharing. It aims to be a standard for one contract to rate another contract or wallet, so I don't think the token terminology is best anymore.